charts/backstage/templates/backstage-deployment.yaml

```diff
@@ -127,12 +127,12 @@ spec:
           volumeMounts:
             {{- range .Values.backstage.extraAppConfig }}
             - name: {{ .configMapRef }}
-              mountPath: "/app/{{ .filename }}"
+              mountPath: "{{ .Values.backstage.installDir }}/{{ .filename }}"
```

Comment:
`.Values.backstage.installDir` is not within the scope of the `range` block. Installing the chart will fail with an error saying  _<.Values.backstage.extraAppConfig>: nil pointer evaluating interface {}.backstage_

installDir needs to be [assigned to a variable](https://helm.sh/docs/chart_template_guide/variables/#helm) outside of the range block